Sources/Datadog/Tracing/AutoInstrumentation/URLSessionTracingHandler.swift

```diff
@@ -55,20 +55,17 @@ internal class URLSessionTracingHandler: URLSessionInterceptionHandler {
         span.setTag(key: OTTags.httpMethod, value: method)
 
         if let error = resourceCompletion.error {
-            span.setTag(key: OTTags.error, value: true)
-
-            let dderror = DDError(error: error)
-            span.setTag(key: DDTags.errorType, value: dderror.title)
-            span.setTag(key: DDTags.errorMessage, value: dderror.message)
-            span.setTag(key: DDTags.errorStack, value: dderror.details)
+            span.setError(error)
         }
 
-        if let httpResponseStatusCode = resourceCompletion.httpResponse?.statusCode {
-            span.setTag(key: OTTags.httpStatusCode, value: httpResponseStatusCode)
-
-            if httpResponseStatusCode >= 400 && httpResponseStatusCode < 500 {
-                span.setTag(key: OTTags.error, value: true)
-                if httpResponseStatusCode == 404 {
+        if let httpResponse = resourceCompletion.httpResponse {
+            let httpStatusCode = httpResponse.statusCode
+            span.setTag(key: OTTags.httpStatusCode, value: httpStatusCode)
+            if httpStatusCode >= 400 && httpStatusCode < 500 {
+                if let error = httpResponse.asError() {
+                    span.setError(error)
```

Comment:
As we're capturing the `file` and `line` automatically as a part of the `setError()` API, this will be always giving `Datadog/URLSessionTracingHandler.swift` for the customers (as on the picture below). Do we consider this as a problem? IMO this could be slightly misleading (especially when aggregating "all errors" by their "stack", all auto instrumented network errors will lead to our SDK's stack trace). Is it possible to not pass this information?

To seed this span, enable `TracingURLSessionScenario` in `Example's` app `ENV`, then run it:

<img width="865" alt="Screenshot 2021-02-01 at 11 06 01" src="https://user-images.githubusercontent.com/2358722/106444102-96e1c480-647d-11eb-9704-9bdc2cd06f48.png">